tests/cli/conftest.py

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8 -*-
+# pylint: disable=redefined-outer-name
+"""Fixtures for the command line interface."""
+import pytest
+
+
+def mock_launch_process(*_, **__):
+    """Mock the :meth:`~aiida_quantumespresso.cli.utilslaunch.launch_process` to be a no-op."""
+    return
+
+
+@pytest.fixture
+def run_cli_command():
+    """Run a `click` command with the given options.
+
+    The call will raise if the command triggered an exception or the exit code returned is non-zero.
+    """
+
+    def _run_cli_command(command, options=None, raises=None):
+        """Run the command and check the result.
+
+        :param command: the command to invoke
+        :param options: the list of command line options to pass to the command invocation
+        :param raises: optionally an exception class that is expected to be raised
+        """
+        import traceback
+        from click.testing import CliRunner
+
+        runner = CliRunner()
+        result = runner.invoke(command, options or [])
+
+        if raises is not None:
+            assert result.exception is not None, result.output
```

Comment:
Shouldn't there be a way to check for the error message that is printed here?

Reply:
We could add this later yeah, where you can specify a `match` argument that can match the error like in `pytest.raises`.